galaxy_ng/app/api/v3/serializers/collection.py

```diff
@@ -105,10 +105,11 @@ def get_download_url(self, obj):
         """
         Get artifact download URL.
         """
-        host = settings.CONTENT_ORIGIN.strip("/")
-        prefix = settings.CONTENT_PATH_PREFIX.strip("/")
+
+        host = settings.ANSIBLE_API_HOSTNAME.strip("/")
+        prefix = settings.GALAXY_API_PATH_PREFIX.strip("/")
```

Comment:
What was the motivation for changing these two lines?

Reply:
Actually I realize now you want the not-content-app to hand out the data. That makes the `ANSIBLE_API_HOSTNAME` change look good. I don't think I understand what `GALAXY_API_PATH_PREFIX` is since it's not a pulp_ansible setting https://docs.pulpproject.org/pulp_ansible/settings.html

Reply:
That points the download URL to the galaxy apis instead of the content app